pkg/skaffold/deploy/helm.go

```diff
@@ -506,6 +507,10 @@ func installArgs(r latest.HelmRelease, builds []build.Artifact, valuesSet map[st
 		args = append(args, "--namespace", o.namespace)
 	}
 
+	if r.CreateNamespace && o.helmVersion.GE(helm32Version) && !o.upgrade {
```

Comment:
can we fail fast with an appropriate error message when `CreateNamespace` is specified but the min required helm version is not met? You'd have to make it `*bool` in `config.go`